package.json

```diff
@@ -73,35 +78,33 @@
     "cross-env": "^7.0.3",
     "document-drive": "0.0.30",
     "document-model": "^1.0.30",
-    "eslint": "^8.56.0",
+    "eslint": "^9.10.0",
     "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-import": "^2.29.1",
```

Comment:
I removed a bunch of plugins which were installed but not used in the eslint config